WordOfTheDay.Repository/WordsRepository.cs

```diff
@@ -64,23 +60,21 @@ public async Task PostWord(Word word)
         }
         public Task<bool> IsAlreadyExist(Word word)
         {
-            var time = DateTime.Today.ToUniversalTime();
-
             var exist = _context.Words
-                .AnyAsync(w => w.Email == word.Email && w.AddTime > time);
+                .LaterThan()
+                .AnyAsync(w => w.Email == word.Email);
 
             return exist;
         }
         public async Task<WordCount> UserWord(string email)
         {
-            var time = DateTime.Today.ToUniversalTime();
-
             var word = await _context.Words
-                .Where(word => word.AddTime > time)
+                .LaterThan()
                 .SingleOrDefaultAsync(w=>w.Email==email);
 
             var userWordAmount = await _context.Words
-                .Where(w => w.AddTime > time && w.Text == word.Text)
+                .LaterThan()
+                .Where(w => w.Text == word.Text)
```

Comment:
Let's put this Where statement to extension also. e.g. .ByText(text)

Reply:
done